frontend/lib/identification/no_card_view.dart

```diff
@@ -1,55 +1,60 @@
-                "Noch keine Ehrenamtskarte hinterlegt",
-                textAlign: TextAlign.start,
-                style: Theme.of(context)
-                    .textTheme
-                    .headline6
-                    .merge(TextStyle(color: Colors.white70)),
-              )),
-        ),
-        SizedBox(height: 24),
-        Flexible(
-            fit: FlexFit.loose,
-            child: Center(
-                child: ElevatedButton(
-                    style: ElevatedButton.styleFrom(
-                      shape: RoundedRectangleBorder(
-                        borderRadius: BorderRadius.circular(32.0)),
-                      padding:
-                        EdgeInsets.symmetric(vertical: 12, horizontal: 24),
-                    ),
-                    onPressed: onOpenQrScanner,
-                    child: Text(
-                      "Code einscannen",
-                      style: Theme.of(context)
-                          .textTheme
-                          .headline6
-                          .merge(TextStyle(color: Colors.white, fontSize: 20)),
-                    ))))
-      ],
+    return Center(
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.all(30.0),
+            child: Icon(Icons.contact_support_outlined,
+                size: 100, color: Theme.of(context).accentColor),
+          ),
+          SizedBox(height: 60),
+          ConstrainedBox(
+              constraints: BoxConstraints(maxWidth: 300),
+              child: Column(children: [
+                Text("Sie sind ehrenamtlich engagiert …",
+                    style: Theme.of(context).textTheme.headline6,
+                    textAlign: TextAlign.center),
+                SizedBox(height: 24),
+                Text(
+                    "… und haben bereits einen Aktivierungscode"
+                    " für die digitale Ehrenamtskarte?",
+                    textAlign: TextAlign.center),
+                TextButton(
+                  onPressed: startActivateQrCode,
+                  child: Text("Jetzt Aktivierungscode einscannen"),
+                )
+              ])),
+          Divider(height: 80),
+          ConstrainedBox(
+              constraints: BoxConstraints(maxWidth: 300),
+              child: Column(children: [
+                Text(
+                  "Sie arbeiten bei einer Akzeptanzstelle …",
+                  style: Theme.of(context).textTheme.headline6,
+                  textAlign: TextAlign.center,
+                ),
+                SizedBox(height: 24),
+                Text(
+                    "… und möchten eine Ihnen gezeigte digitale Ehrenamtskarte"
+                    " auf Echtheit prüfen?",
+                    textAlign: TextAlign.center),
+                TextButton(
+                  onPressed: startVerification,
+                  child: Text("Jetzt Ehrenamtskarte validieren"),
+                )
+              ]))
+        ],
```

Comment:
Looks a bit strange in landscape mode (much to scroll for that few lines of text). Maybe it can be improved easily with a `Flex`, so that both "blocks" are next to each other on landscape mode?